pymc3/distributions/distribution.py

```diff
@@ -244,6 +248,7 @@ def __new__(
             functools.partial(str_for_dist, formatting="latex"), rv_out
         )
 
+        rv_out.logp = _nice_logp_attribute_error
```

Comment:
I don't think you need this one?

Reply:
We need it because of the resizing that can happen a few lines above.
That's why in the test case I'm adding a dim that triggers a resize _after_ the `.dist()` line.

Reply:
Maybe mention that in the test?